packages/spotlight/src/spotlight.js

```diff
@@ -359,7 +359,6 @@ const Spotlight = (function () {
 		const pointerHandled = notifyKeyDown(keyCode, handlePointerHide);
 
 		if (pointerHandled) {
-			_pointerMoveDuringKeyPress = true;
```

Comment:
I would have to go back and look, but I _think_ this flag is supposed to be set in the block below.  The intention being that if you were holding a key down and moved the pointer, it's not supposed to activate pointer mode.  It seems when we split the handling before we removed this from the conditional.  If this is the case we don't wish to set this anymore then we should remove the logic associated with it, since it's no longer possible to have this flag set.  We can further simplify by recombining this `if` with the one below.